frontend/kesaseteli/employer/src/components/application/steps/step2/accordions/EmploymentAccordions.tsx

```diff
@@ -0,0 +1,37 @@
+import AddNewEmploymentButton from 'kesaseteli/employer/components/application/steps/step2/accordions/AddNewEmploymentButton';
```

Comment:
Represents the list of accordions
![Uploading Screenshot 2021-09-20 at 3.55.04 PM.png…]()